Makefile

```diff
@@ -92,6 +103,6 @@ endif
 	git pull --quiet origin && \
 	git checkout v$(ZMQ_VERSION) && \
 	./autogen.sh && \
-	./configure --quiet --without-docs --prefix=$(ZMQ_BUILD_DIR) \
+	./configure CXXFLAGS=$(CXXFLAGS) --quiet --without-docs --prefix=$(ZMQ_BUILD_DIR) \
 		--enable-drafts=yes --enable-libunwind=no --enable-static=no && \
 	$(MAKE) install
```

Comment:
Don't we want to do something like the following after `$(MAKE) install`?

```
ifeq ($(OS),Windows_NT)
    cp $(ZMQ_BUILD_DIR)/bin/libzmq.dll $(CURDIR)
endif
```

as per https://github.com/dzop/emacs-zmq/issues/3#issuecomment-461939172. The `emacs-zmq.dll` will eventually end up in `$(CURDIR)`.